substrate/frame/support/procedural/src/pallet/parse/validate_unsigned.rs

```diff
@@ -19,15 +19,10 @@ use super::helper;
 use syn::spanned::Spanned;
 
 /// The definition of the pallet validate unsigned implementation.
-pub struct ValidateUnsignedDef {
-	/// The index of validate unsigned item in pallet module.
-	pub index: usize,
-	/// A set of usage of instance, must be check for consistency with config.
-	pub instances: Vec<helper::InstanceUsage>,
-}
+pub struct ValidateUnsignedDef {}
 
 impl ValidateUnsignedDef {
-	pub fn try_from(index: usize, item: &mut syn::Item) -> syn::Result<Self> {
+	pub fn try_from(_index: usize, item: &mut syn::Item) -> syn::Result<Self> {
```

Comment:
There are still more of these unused params?

Reply:
> There are still more of these unused params?

I considered the `index` as part of the standard API across all pallet components, but sure let me remove that as well.